### PR TITLE
Add User Session blueprint and Create method

### LINKED ISSUE #22: Add route for admins to create user sessions
Follow up for https://github.com/hoodiehq/account-json-api/issues/14

As an admin, I would like to be able to create a user session, so I can debug an account without messing with passwords.

I would suggest the route `POST /account/{id}/sessions` for that. Without any body (does JSON API allow that?).

The response would look like this:

``` json
{
  "links": {
    "self": "https://example.com/accounts/abc1234/sessions/sessionid123"
  },
  "data": {
    "id": "sessionid123",
    "type": "session",
    "relationships": {
      "account": {
        "links": {
          "related": "https://example.com/accounts/abc1234"
        },
        "data": {
          "id": "abc1234",
          "type": "account"
        }
      }
    }
  }
}
```

This implies that we would also need a `GET /accounts/{id}/sessions/{id}` and a `DELETE /accounts/{id}/sessions/{id}` route, which works for me.

Any objections / thoughts?

### PR DESCRIPTION
**Pull request by barretodaniel**
closes #22

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1253,3 +1253,55 @@ All custom user data is stored in the profile.
                 "detail": "'type' and 'id' provided don't match any existing document"
             }]
         }
+
+## User Session [/accounts/{id}/sessions]
+
+
```
Comment (barretodaniel) [inline apiary.apib]: Not sure what description should be put here

File: apiary.apib [hunk 2]
```diff
@@ -1253,3 +1253,55 @@ All custom user data is stored in the profile.
...
++ Response 201 (application/vnd.api+json)
+
+        {
+            "links": {
+                "self": "https://example.com/accounts/abcd123/sessions/session123"
+            },
+            "data": {
+                id: 'session123',
+                account: {
```
Comment (barretodaniel) [inline apiary.apib]: Looking at other blueprints, I should probably add a type. would that be `session`?

File: apiary.apib (hunk 1 above)
Comment (gr2m) [reply apiary.apib]: Something like this maybe? 

> Admins can manage sessions for all user accounts

File: apiary.apib (hunk 2 above)
Comment (gr2m) [reply apiary.apib]: yes

Comment (gr2m) [reply apiary.apib]: Also it’s `"relationships": {"account": ...`. And make sure all properties and values are wrapped in `"` because it's JSON

**Review by gr2m — CHANGES_REQUESTED** [review]

File: apiary.apib [hunk 3]
```diff
@@ -1278,10 +1278,22 @@ All custom user data is stored in the profile.
...
-                    id: 'account456',
-                    username: 'pat@example.com'
+                "id": "session123",
+                "type": "session",
+                "account": {
+                    "id": "abcd123",
+                    "username": "pat@example.com"
+                }
+            },
+            "relationships": {
```
Comment (barretodaniel) [inline apiary.apib]: Hoping I did this one right

**Review by gr2m — APPROVED** [review]
LGTM 👍

**Pull request merged**